File: CodeParser.Test/Examples/Helloworld.cpp
#include <iostream.h>
#include <string>

using namespace std;

main()
{
    cout << "Hello World!";
    return 0;
}

class HelloWorld
{
public:
	string owner, target;
	int age;
	

public:
	void sayHello(string word)
	{
		cout << word;
	}

	void test()
	{
		cout << "Test";
	}
};

File: CodeParser.Test/Examples/HelloWorld.php
<?
namespace demo;
use ArrayObject;

$count =0;

class HelloWorld {
    public string $owner;
    public string $target;

	function sayHello(?string $word)
	{
		echo $word;
	}
}

function test()
{
   $helloWorld =new HelloWorld();
   $helloWorld->sayHello();
}

?>